File: backend/DataAccess/DaoGenerated/システム共有/tm_afstaff_subDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             事業従事者（担当者）サブマスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_afstaff_subDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_afstaff_sub";
		public string staffid { get; set; }                       //事業従事者ID
		public string jissijigyo { get; set; }                    //医療機関・事業従事者（担当者）事業コード
    }
}

File: backend/JbdGjsService/Db/DaoGenerated/システム共有/tm_afstaff_subDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             事業従事者（担当者）サブマスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tm_afstaff_subDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_afstaff_sub"
        Public Property staffid As String                       '事業従事者ID
        Public Property jissijigyo As String                    '医療機関・事業従事者（担当者）事業コード
    End Class
End Namespace
